docs/installation.md

```diff
@@ -19,7 +19,8 @@ pipx works on macOS, linux, and Windows.
 ```
 brew install pipx
 pipx ensurepath
-sudo pipx ensurepath --global # optional to allow pipx actions in global scope. See "Global installation" section below.
+sudo pipx ensurepath --global # optional to allow pipx actions in global scope. See "Customising your installation" section below.
+sudo pipx ensurepath --prepend # optional to prepend the pipx bin directory to PATH instead of appending it. See "Customising your installation" section below.
```

Comment:
I'd prefer having a comment after line 22: `pipx ensurepath # optional with --prepend, to prepend the pipx bin directory to PATH instead of appending it. See "Customising your installation" section below.`, instead of having it as a separate line. 

So far the install instructions can just be executed top to bottom and everything is fine. If we add your line, and the user goes top to bottom, they would get a warning for prepending, as pipx is already in PATH. For a user that does not really read the description and just copies the instructions, this would be confusing IMO.

Same for the other install instructions.

Reply:
Makes sense. I'll move both commands, `sudo pipx ensurepath --global` and `sudo pipx ensurepath --prepend` to a code block of their own so that they aren't inadvertently copied and run by a user on their system.

Reply:
Addressed in 78c28930f9f62b0f26abf63fb2246bad34f8a146.